tests/queries/0_stateless/02030_client_unknown_database.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+CLICKHOUSE_DATABASE=no_such_database_could_exist
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+$CLICKHOUSE_CLIENT -q "SELECT 1" |& grep -q "Database $CLICKHOUSE_DATABASE doesn't exist."
```

Comment:
Let's match by exception code, not by message.

Reply:
Good idea, updated.